calendar_dst_bug_fix/tests/test_calendar_recurrent_event_with_dst.py

```diff
@@ -0,0 +1,65 @@
+
+# -*- coding: utf-8 -*-
+
+from odoo.tests import common
+from datetime import datetime
+import pytz
+
+
+class TestRecurrentEventWithDst(common.TransactionCase):
+
+    def setUp(self):
+        super(TestRecurrentEventWithDst, self).setUp()
+        self.CalendarEvent = self.env['calendar.event']
```

Comment:
```suggestion
        self.calendar_event = self.env['calendar.event']
```

https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#variable-names